jena-sdb/assembly.xml

```diff
@@ -41,11 +41,11 @@
 
   <files>
     <file>
-      <source>dist/LICENSE</source>
+      <source>LICENSE</source>
```

Comment:
Sorry, I'm confused now. Which `dist/LICENSE` should this be? There is no `jena-sdb/dist` directory nowadays. There is `jena-fuseki1/dist` and `apache-jena-fuseki/dist` currently.